_overviews/FAQ/index.md

```diff
@@ -110,28 +110,56 @@ One would hope so, but doing it that way was tried and it proved
 impossible.  [This SO question](https://stackoverflow.com/questions/11167430/why-are-primitive-types-such-as-int-erased-to-object-in-scala)
 sadly lacks a concise explanation, but it does link to past discussions.
 
-### Difference between methods and functions
+### What's the difference between methods and functions?
+
+For example, how does a method such as:
+
+    def square(x: Int): Int = x * x
+
+differ from a function value such as:
+
+    val square: Int => Int = x => x * x
 
 [complete answer on Stack Overflow](https://stackoverflow.com/a/2530007/4111404)
 
 [summary with practical differences](https://tpolecat.github.io/2014/06/09/methods-functions.html)
 
-### Difference between classes and types
+### What's the difference between types and classes?
+
+Types are primarily a compile-time concept. At compile time,
+every expression is assigned a type by the compiler.
+
+Classes are primarily a runtime concept and are platform-dependent.
+At runtime on the JVM, every value is either a primitive value
+or an instance of exactly one class.
+
+Some type information exists only at compile time, for multiple
+reasons, most notoriously [type
+erasure](https://en.wikipedia.org/wiki/Type_erasure).
```

Comment:
Big +1 on this, probably on the very first JVM things I learned and it was due **Scala**.